ITEM.tsv

```diff
@@ -5858,7 +5858,7 @@ ITEM_20150729_005857	Since Medzio Diena, many equipment crafted from new materia
 ITEM_20150729_005858	Standard armor given to Kingdom Guards prior to Medzio Diena.
 ITEM_20150729_005859	Since its inauguration prior to Medzio Diena, the Cryomancer Master has stopped the tradition of awarding this pendant to people. Now this pendant must be either obtained or crafted by users themselves.
 ITEM_20150729_005860	Many new magical items that uses the power of monsters have been invented since Medzio Diena.
-ITEM_20150729_005861	- Electric impact effect to enemies with damage 405 by 15% random when receive the electric attribute attack
+ITEM_20150729_005861	- 15% chance to deal 405 damage to enemies when recieving lightning-type attacks
```

Comment:
It is not emphasized in brackets in the original, because it works that way better in Korean.
But in English it doesn't work that well. 

"전기 충격 효과" 

I've given other effects also the "[XXX Effect]" treatment. It's clear.